packages/miniflare/src/workers/core/entry.worker.ts

```diff
@@ -144,7 +144,6 @@ function maybeLogRequest(
       headers: { "Content-Type": "application/json" },
       body: JSON.stringify({
         status: response.status,
-        statusText: response.statusText,
```

Comment:
Is there any scenario where `response.statusText` is different to what would be retrieved from `http.STATUS_CODES`? I.e. should we try using this first and fall back to the lookup if not available?